vizro-ai/changelog.d/20240415_223857_nadija_ratkusic_graca_refactor_vizroai_class.md

```diff
@@ -0,0 +1,45 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Removed
+
+- Removed `temperature` argument of `VizroAI` class. ([#423](https://github.com/mckinsey/vizro/pull/423))
```

Comment:
Do we not also remove the `model_name` argument? We should also provide a guide (or link to a guide) with what the user should do now.